src/ProjectTemplates/Web.ProjectTemplates/content/RazorComponentsWeb-CSharp/Startup.cs

```diff
@@ -45,16 +42,16 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
 
 #if (RequiresHttps)
             app.UseHttpsRedirection();
+
 #endif
             app.UseStaticFiles();
 
             app.UseRouting();
 
             app.UseEndpoints(endpoints =>
             {
-                endpoints.MapRazorPages();
-                endpoints.MapComponentHub();
-                endpoints.MapFallbackToPage("/Host");
+                endpoints.MapComponentHub<App>("app");
```

Comment:
If you do this, this will throw away the data from the prerendered circuit and will likely cause wreck havoc on the client as two renderers are trying to render on the same node.

Reply:
Ok, I’ll revert this bit then. 

I really don’t want the suggested big comment in the template, so we’ll just have to accept that disabling prerendering is now a nonobvious process. It should get a bit better when there’s a way of emitting the placeholder without prerendering.